bootstrap/mocks/things.go

```diff
@@ -230,3 +231,47 @@ func findIndex(list []string, val string) int {
 
 	return -1
 }
+
+func (svc *mainfluxThings) CreateGroup(ctx context.Context, token string, group groups.Group) (groups.Group, error) {
+	panic("not implemented")
```

Comment:
No need to panic and break the whole service

Reply:
Also, please open an issue for this and paste the issue reference as a comment to this PR.

Reply:
This is a mock, and while it isn't used it will not break anything similar to other places

Reply:
Comment resoled by issue #1271.